src/database/votePost.js

```diff
@@ -2,9 +2,17 @@ import { db } from "database";
 import firebase from "firebase";
 
 export const voteForOption = ({ optionName, postId, userId }) => {
+  const increment = firebase.firestore.FieldValue.increment(1);
+
   db.collection("posts")
     .doc(postId)
     .update({
       [optionName]: firebase.firestore.FieldValue.arrayUnion(userId),
+      total_votes: increment,
+    });
+  db.collection("users")
```

Comment:
Are you just doing two database calls one after the other inside the same function here? Looks like an anti-pattern.  You want to split these out into different functions and then fire them separately. Remember one functions should ideally do one thing

Reply:
kk

Reply:
split up
done